src/Command/Branch/BranchChangelogCommand.php

```diff
@@ -25,16 +27,31 @@ protected function configure()
     {
         $this
             ->setName('branch:changelog')
-            ->setDescription('Reports what got fixed or closed since last release on current branch')
+            ->setDescription('Reports what got fixed or closed since last release on the branch')
```

Comment:
Reports what has been fixed or closed since the last release on the given branch